staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -136,7 +136,7 @@ func (o WardleServerOptions) RunWardleServer(stopCh <-chan struct{}) error {
 	}
 
 	server.GenericAPIServer.AddPostStartHook("start-sample-server-informers", func(context genericapiserver.PostStartHookContext) error {
-		config.GenericConfig.SharedInformerFactory.Start(context.StopCh)
```

Comment:
I think this is a bug. config.GenericConfig.SharedInformerFactory is not set and is nil.

Reply:
don't we run this server in tests?